qiskit_experiments/library/calibration/fine_drag.py

```diff
@@ -0,0 +1,271 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fine DRAG calibration experiment."""
+
+from typing import Optional, List
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Gate
+from qiskit.providers import Backend
+
+from qiskit_experiments.framework import BaseExperiment, Options
+from qiskit_experiments.library.calibration.analysis.fine_drag_analysis import (
+    FineDragAnalysis,
+)
+
+
+class FineDrag(BaseExperiment):
+    r"""Fine DRAG Calibration experiment.
+
+    # section: overview
+
+        The class :class:`FineDrag` runs fine DRAG calibration experiments (see :class:`DragCal`
```

Comment:
Now you can use see_also section.
https://github.com/Qiskit/qiskit-experiments/blob/023b5ed838202b4d80aac07b8758ff6d37cb73f2/qiskit_experiments/library/characterization/cr_hamiltonian_analysis.py#L129-L130